### ISSUE TITLE
EC Bestiary: Qurashith is missing traits on attacks

### ISSUE THREAD

**Issue by cswendrowski** [foundryvtt/pf2e#1244]
In GitLab by @Ustin on Mar 29, 2021, 15:29

- Jaws should have _chaotic, evil, magical, reach 15 feet_
- Claw should have _agile, chaotic, evil, magical, reach 15 feet_
- Spittle should have _range 60 feet_

Link: https://pf2.easytool.es/index.php?id=6600&name=qurashith
